app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
+  },
+  "queue": {
+    "message": "Cola"
+  },
+  "rejectAll": {
+    "message": "Rechazar a Todos"
+  },
+  "rejectTxsN": {
+    "message": "Rechazar a transacciones de $1"
+  },
+  "rejectTxsDescription": {
+    "message": "Está al punto de rechazar transacciones de $1 en lote."
+  },
+  "reset": {
+    "message": "Reiniciar"
+  },
+  "resetAccountDescription": {
+    "message": "Reiniciar tu cuenta borrará tu historial de transacciones."
+  },
+  "restoreAccountWithSeed": {
+    "message": "Restaurar tu Cuenta con Frase Semilla"
+  },
+  "requestsAwaitingAcknowledgement": {
+    "message": "peticiones pendientes de reconocimiento"
+  },
+  "restore": {
+    "message": "Restaurar"
+  },
+  "revealSeedWordsTitle": {
+    "message": "Frase Semilla"
+  },
+  "revealSeedWordsDescription": {
+    "message": "Si en algún momento cambias de navegador o de ordenador, necesitarás esta frase semilla para acceder a tus cuentas. Guárdatela en un lugar seguro y secreto."
+  },
+  "revealSeedWordsWarningTitle": {
+    "message": "NO compartas esta frase con nadie!"
+  },
+  "remove": {
+    "message": "borrar"
+  },
+  "removeAccount": {
+    "message": "Borrar cuenta"
+  },
+  "removeAccountDescription": {
+    "message": "Se borrará esta cuenta de tu monedero. Por favor, asegúrate de tener la frase semilla o clave personal original para esta cuenta importada antes de seguir adelante. Podrás importar o crear cuentas de nuevo del menu desplegable de cuentas."
+  },
+  "readyToConnect": {
+    "message": "¿Listo/a para conectar?"
+  },
+  "rpc": {
+    "message": "RPC Personalizado"
+  },
+  "slow": {
+    "message": "Lento"
+  },
+  "slower": {
+    "message": "Más Lento"
+  },
+  "saveAsCsvFile": {
+    "message": "Guardar como archivo CSV"
+  },
+  "scanInstructions": {
+    "message": "Coloque el código QR delante de tu cámara"
+  },
+  "scanQrCode": {
+    "message": "Escanear código QR"
+  },
+  "searchResults": {
+    "message": "Resultados de la Búsqueda"
+  },
+  "secondsShorthand": {
+    "message": "Seg"
+  },
+  "selectLocale": {
+    "message": "Seleccionar Local"
+  },
+  "sendAmount": {
+    "message": "Mandar Cantidad"
+  },
+  "sentEther": {
+    "message": "se mandó ether"
+  },
+  "sentTokens": {
+    "message": "se mandaron tokens"
+  },
+  "separateEachWord": {
+    "message": "Separar a cada palabra con un sólo espacio"
+  },
+  "searchTokens": {
+    "message": "Buscar Tokens"
+  },
+  "selectAnAddress": {
+    "message": "Seleccionar una Dirección"
+  },
+  "selectAnAccount": {
+    "message": "Seleccionar una Cuenta"
+  },
+  "selectAnAccountHelp": {
+    "message": "Seleccionar la cuenta que quiere ver en MetaMask"
+  },
+  "selectAHigherGasFee": {
+    "message": "Seleccione una comisión de gas más elevada para agilizar el procesamiento de tu transacción.*"
+  },
+  "selectHdPath": {
+    "message": "Seleccionar Ruta al disco duro"
```

Comment:
"Selecciona el HD (Hierarchical Deterministic) path"

Esto es un concepto de las hardware wallets. Esta traducción si que la tienes que corregir!

Reply:
Thank you!! Me dio mala espina.

OK, I found it in this glossary: https://academy.bit2me.com/diccionario-crypto/

Under 'master keys' they talk about them being 'jerárquicas deterministas'. So, that seems to be the emerging term...? I'll roll with it.